app/views/quotes/index.html.erb

```diff
@@ -9,7 +9,7 @@
       </footer>
     </blockquote>
     <div class="created">
-      <%= time_ago_in_words(quote.created_at) %> ago
+      <%= time_ago_in_words(quote.created_at) + " ago" %>
```

Comment:
Can we revert this? We don't need to do the string concatenation, the html rendering for `ago` should work just fine :)